web/Views/Property/ViewProperty.cshtml

```diff
@@ -26,6 +26,13 @@
     </div>
 
     <div class="col-md d-flex flex-column pb-2">
+        <div class="d-flex">
+            <h3 class="font-h3 d-inline-block">@(User.IsInRole("Landlord") || User.IsInRole("Admin") ? "Tenant preferences" : "Suitable for")</h3>
+            @if (User.IsInRole("Landlord") || User.IsInRole("Admin"))

```

Comment:
I've seen this in a lot of different places, and I think this is an example where adding an extension to the ClaimsPrincipal class (what User is) that is something like `IsLandlordOrAdmin` :)